Modules/_asynciomodule.c

```diff
@@ -3844,6 +3861,7 @@ module_init(asyncio_state *state)
 {
     PyObject *module = NULL;
 
+    PyMutex_Lock(&state->mutex);
```

Comment:
No locking in `module_init`. This is called before the module is visible to other threads.